<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;AkMGRH47eCl7ImA9WhVSFUU.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/2602/comments/full/3</ns0:id><ns0:author>
			<ns0:name>bugdro...@chromium.org</ns0:name><ns0:uri>/u/101788990881243245974/</ns0:uri></ns0:author><ns0:content type="html">The following revision refers to this bug:
    http://src.chromium.org/viewvc/native_client?view=rev&amp;revision=8002

------------------------------------------------------------------------
r8002 | mseaborn@chromium.org | Mon Mar 12 14:37:32 PDT 2012

Changed paths:
 M http://src.chromium.org/viewvc/native_client/trunk/src/native_client/tests/inbrowser_test_runner/nacl.scons?r1=8002&amp;r2=8001&amp;pathrev=8002

Exception handling: Split test so that it can be separately enabled/disabled

This enables the test for exception handling for NaCl-in-Chromium but
gives it a separate name, 'run_inbrowser_exception_test', so that it
can be disabled from Chromium's buildbot_nacl_integration.py script
until exception handling support is implemented on the Chromium side
on all platforms.

BUG=http://code.google.com/p/nativeclient/issues/detail?id=2602
TEST=none

Review URL: https://chromiumcodereview.appspot.com/9664028
------------------------------------------------------------------------</ns0:content><ns0:updated>2012-03-12T21:40:25.000Z</ns0:updated><ns0:published>2012-03-12T21:40:25.000Z</ns0:published><ns2:updates /><ns0:title>Comment 3 by bugdro...@chromium.org</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=2602#c3" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/2602/comments/full/3" rel="self" type="application/atom+xml" /></ns0:entry>